src/libsync/pushnotifications.h

```diff
@@ -0,0 +1,68 @@
+#ifndef PUSHNOTIFICATIONS_H
```

Comment:
Missing copyright header (like for every new files in this PR AFAICT).

Reply:
How do I do them? Just copy them from the other files? Who should I pick as author? Me?